### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DzmitryU**
Added article and comment list

### CODE REVIEW

File: src/components/Article.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    handleButtonClick = () => {
+        this.setState({
+            isOpen: !this.state.isOpen
+        })
+    };
+
+    render() {
+        let body = '';
```
Comment (sajdre) [inline src/components/Article.js]: use const where it's possible (i.e. everywhere)

File: src/components/CommentList.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    handleButtonClick = () => {
+        this.setState({
+            isOpen: !this.state.isOpen
+        });
+    };
+
+    render() {
+        let body = '';
```
Comment (sajdre) [inline src/components/CommentList.js]: use const where it's possible (i.e. everywhere)

File: src/components/Comment.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
```
Comment (sajdre) [inline src/components/Comment.js:2]: functional component can be used (there is no state)

File: src/components/Article.js (hunk 1 above)
Comment (DzmitryU) [reply src/components/Article.js]: Done.

File: src/components/CommentList.js (hunk 2 above)
Comment (DzmitryU) [reply src/components/CommentList.js]: Do you think that let is worse then if-branching and double return in method?

File: src/components/Comment.js (hunk 3 above)
Comment (DzmitryU) [reply src/components/Comment.js:2]: Done

**Pull request merged**